tools/json2bin/consts/moves.py

```diff
@@ -0,0 +1,474 @@
+from enum import Enum, auto
+
+class Move(Enum):
+    MOVE_0000 = 0
```

Comment:
Something something maybe move constants should be defined from the jsons alone? idk

Reply:
That's what pfero and I have been trying to figure out

Reply:
Probably not necessary yet.

Reply:
Yeah, calling this one out of scope for now; we can revisit it as a problem later if it comes to it.